Bottom: c076dedd7fc3859a921947137ceb8b57fa008eb2
Top:    00e1df4bcbc33b38df0e85b87fbbabd2da64d772
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-09 17:16:01 -0700

Refresh of build-preload

---

diff --git a/Makefile.am b/Makefile.am
index b734d45..901e8ee 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,12 +1,12 @@
 INCLUDES = -I$(srcdir)/include
 
-lib_LTLIBRARIES = src/librdmacm.la src/rspreload.la
+lib_LTLIBRARIES = src/librdmacm.la src/librspreload.la
 
 ACLOCAL_AMFLAGS = -I config
 AM_CFLAGS = -g -Wall -D_GNU_SOURCE -DSYSCONFDIR=\"$(sysconfdir)\" -DRDMADIR=\"@rdmadir@\"
 
 src_librdmacm_la_CFLAGS = $(AM_CFLAGS)
-src_rspreload_la_CFLAGS = $(AM_CFLAGS) -shared -fPIC
+src_librspreload_la_CFLAGS = $(AM_CFLAGS)
 
 if HAVE_LD_VERSION_SCRIPT
     librdmacm_version_script = -Wl,--version-script=$(srcdir)/src/librdmacm.map
@@ -20,7 +20,8 @@ src_librdmacm_la_LDFLAGS = -version-info 1 -export-dynamic \
 			   $(librdmacm_version_script)
 src_librdmacm_la_DEPENDENCIES =  $(srcdir)/src/librdmacm.map
 
-src_rspreload_la_SOURCES = src/preload.c src/indexer.c
+src_librspreload_la_SOURCES = src/preload.c src/indexer.c
+src_librspreload_la_LIBADD = $(top_builddir)/src/librdmacm.la
 
 bin_PROGRAMS = examples/ucmatose examples/rping examples/udaddy examples/mckey \
 	       examples/rdma_client examples/rdma_server examples/rdma_xclient \
